### PR TITLE
feat(tracing): stdout exporter sample

### PR DESCRIPTION
**Pull request by ev1lQuark**

### CODE REVIEW

**Comment by chickenlj** [comment]
Could you please give a simple README instructing what kind of metrics will generate and how the users can use them?

**Comment by FinalT** [comment]
plz add test directories to `start_integrate_test.sh`

**Review by FinalT — COMMENTED** [review]
plz add test directories to `start_integrate_test.sh`

File: start_integrate_test.sh [hunk 1]
```diff
@@ -35,6 +35,9 @@ array+=("compatibility/error/triple/pb")
 # metrics
 array+=("compatibility/metrics")
 
+# tracing
+array+=("compatibility/otel/tracing/stdout")
+
```
Comment (FinalT) [inline start_integrate_test.sh:40]:
```suggestion
# tracing
array+=("otel/tracing/stdout")

```

File: otel/tracing/stdout/README.md
Comment (FinalT) [inline otel/tracing/stdout/README.md:1]: plz briefly describe how to set

File: start_integrate_test.sh (hunk 1 above)
Comment (ev1lQuark) [reply start_integrate_test.sh]: My bad

File: otel/tracing/stdout/README.md
Comment (ev1lQuark) [reply otel/tracing/stdout/README.md:1]: Done

**Review by chickenlj — APPROVED** [review]
LGTM.

**Pull request merged**